client_internals_it_test.go

```diff
@@ -430,10 +430,10 @@ func filterConnectedMembers(ci *hz.ClientInternal) []cluster.MemberInfo {
 }
 
 const (
-	MCGetMemberConfigCodecRequestMessageType  = int32(0x200500)
+	McGetMemberConfigCodecRequestMessageType  = int32(0x200500)
 	MCGetMemberConfigCodecResponseMessageType = int32(0x200501)
 
-	MCGetMemberConfigCodecRequestInitialFrameSize = proto.PartitionIDOffset + proto.IntSizeInBytes
+	McGetMemberConfigCodecRequestInitialFrameSize = proto.PartitionIDOffset + proto.IntSizeInBytes
```

Comment:
Could you change this to:
```
MCGetMemberConfigCodecRequestMessageType
MCGetMemberConfigCodecRequestInitialFrameSize
```
so the changes are completely reverted?

Reply:
Fixed: [73d0b10](https://github.com/hazelcast/hazelcast-go-client/pull/809/commits/73d0b106afdc713ce08dba54e2ba90be63d5c497).